app/src/main/java/io/github/luizgrp/sectionedrecyclerviewadapter/demo/Example9Fragment.java

```diff
@@ -0,0 +1,203 @@
+import android.widget.TextView;
+import android.widget.Toast;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import io.github.luizgrp.sectionedrecyclerviewadapter.CustomViewType;
+import io.github.luizgrp.sectionedrecyclerviewadapter.SectionParameters;
+import io.github.luizgrp.sectionedrecyclerviewadapter.SectionedRecyclerViewAdapter;
+import io.github.luizgrp.sectionedrecyclerviewadapter.StatelessSection;
+
+public class Example9Fragment extends Fragment {
+
+    CustomViewType custom1 = new CustomViewType(ItemViewHolder1.class, R.layout.section_ex9_item1);
+    CustomViewType custom2 = new CustomViewType(ItemViewHolder2.class, R.layout.section_ex9_item2);
+
+    private SectionedRecyclerViewAdapter sectionAdapter;
+
+    @Nullable
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_ex9, container, false);
+
+        sectionAdapter = new SectionedRecyclerViewAdapter(custom1, custom2);
+
+        for(char alphabet = 'A'; alphabet <= 'Z';alphabet++) {
+            List<String> contacts = getContactsWithLetter(alphabet);
+
+            if (contacts.size() > 0) {
+                sectionAdapter.addSection(new ContactsSection(String.valueOf(alphabet), contacts));
+            }
+        }
+
+        RecyclerView recyclerView = (RecyclerView) view.findViewById(R.id.recyclerview);
+        recyclerView.setLayoutManager(new LinearLayoutManager(getContext()));
+        recyclerView.setAdapter(sectionAdapter);
+
+        return view;
+    }
+
+    @Override
+    public void onResume() {
+        super.onResume();
+
+        if (getActivity() instanceof AppCompatActivity) {
+            AppCompatActivity activity = ((AppCompatActivity) getActivity());
+            if (activity.getSupportActionBar() != null)
+                activity.getSupportActionBar().setTitle(R.string.nav_example9);
+        }
+    }
+
+    private List<String> getContactsWithLetter(char letter) {
+        List<String> contacts = new ArrayList<>();
+
+        for (String contact : getResources().getStringArray(R.array.names)) {
+            if (contact.charAt(0) == letter) {
+                contacts.add(contact);
+            }
+        }
+
+        return contacts;
+    }
+
+
+    private class ContactsSection extends StatelessSection {
+
+        String title;
+        List<String> list;
+
+        ContactsSection(String title, List<String> list) {
+            super(new SectionParameters.Builder(R.layout.section_ex1_item)
+                    .headerResourceId(R.layout.section_ex1_header)
+                    .build());
+
+            this.title = title;
+            this.list = list;
+        }
+
+        @Override
+        public int getContentItemsTotal() {
+            return list.size();
+        }
+
+        @Override
+        public RecyclerView.ViewHolder getItemViewHolder(View view) {
+            return new ItemViewHolder1(view);
```

Comment:
Hi @luizgrp , thanks for reviewing this pull request! getItemViewHolder() was previously defined as an abstract method, so I left it as such to keep existing code working. In Example9Fragment, this method is only called if the ViewType does not match one of the CustomViewTypes assigned to the SectionedRecyclerAdapter. Think of this method as providing the default ViewHolder for an Item.